### ISSUE TITLE
Optional arguments have option type

### ISSUE THREAD

**Issue by dsheets** [lpw25/opam-doc-base#28]
Declared:

``` ocaml
val fun_maybe : ?yes:unit -> unit -> int
```

Documented:

``` ocaml
val fun_maybe : ?yes:unit option -> unit -> int 
```

The `cmi` and `cmti` seem to contain those option types (from what I can discern with a hex editor). Perhaps those formats cannot be changed but the representation in opam-doc-base's interface could be fixed so that arrows with `Default` labels do not contain the option type. Is it possible to have an optional argument without using `'a option`?

**Comment by lpw25** [comment]
> Is it possible to have an optional argument without using 'a option?

No, so it should be quite simple to remove the `option` type from the documentation.

**Comment by samoht** [comment]
I've pushed a crappy fix for that in my branch (0bb9fd3da1eafc1f22c7a042a4361d7beefeeebf)
